src/main/java/cloud/fogbow/rcs/api/http/parameters/ExpirationTimeProperty.java

```diff
@@ -0,0 +1,19 @@
+package cloud.fogbow.rcs.api.http.parameters;
+
+import cloud.fogbow.rcs.constants.ApiDocumentation;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+public class ExpirationTimeProperty {
+    @ApiModelProperty(position = 0, example = ApiDocumentation.Model.EXPIRATION_TIME, required = true)
```

Comment:
To make the code more readable, it may be more intuitive to make the _"0"_ value of the _position_ variable a **constant**.